examples/test_util/EclIO.cpp

```diff
@@ -71,6 +71,29 @@ auto EclIO::flipEndian<double>(const double &num) const {
     return value;
 }
 
+std::tuple<const int, const int> EclIO::block_size_data(EIOD::eclArrType arrType){
+
+    switch(arrType) {
+        case EIOD::INTE  : return std::make_tuple(EIOD::sizeOfInte,EIOD::MaxBlockSizeInte);   break;
+        case EIOD::REAL  : return std::make_tuple(EIOD::sizeOfReal,EIOD::MaxBlockSizeReal);   break;
+        case EIOD::DOUB  : return std::make_tuple(EIOD::sizeOfDoub,EIOD::MaxBlockSizeDoub);   break;
+        case EIOD::LOGI  : return std::make_tuple(EIOD::sizeOfLogi,EIOD::MaxBlockSizeLogi);   break;
+        case EIOD::CHAR  : return std::make_tuple(EIOD::sizeOfChar,EIOD::MaxBlockSizeChar);   break;
+        case EIOD::MESS  : throw std::invalid_argument("Type 'MESS' have not assosiated data") ;   break;
+    }
+}
+
+std::string EclIO::trimr(const std::string &str1) const {
+
+    std::string tmpStr=str1;
```

Comment:
```
auto space_pos = str1.find(' ');
return str1.substr(0, space_pos);
```

Reply:
I agree that the function trimr appears to be longer than what I'm expected. But, what you suggesting assumes that there will be no spaces in what we should store. 

I have just checked with the commercial simulator, and it is possible to use a well name that includes space, for example like  this. 

WELSPECS
   'PROD 3'   'GRP1'    ...   /
/
In this case what you suggest simplification will return "PROD" instead of "PROD 3".

Reply:
Well use a rfind variety

Reply:
The function `string::find_last_of()` should do the trick:
```
auto last_space = str1.find_last_of(' ');
return str.substr(0, last_space);
```
but your solution is perfectly OK.